v0/destinations/appcues/transform.js

```diff
@@ -0,0 +1,184 @@
+const get = require("get-value");
+const set = require("set-value");
+const {
+  EventType
+} = require("../../../constants");
+
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  getValueFromMessage,
+  removeUndefinedAndNullValues,
+  removeUndefinedAndNullAndEmptyValues,
+} = require("../../util");
+
+const {
+  ConfigCategory,
+  mappingConfig,
+  getEndpoint,
+} = require("./config");
+
+const {
+  utcTimeToEpoch,
+} = require("./utils");
+
+function buildResponse(message, payload, endpoint) {
+  const response = defaultRequestConfig();
+  response.endpoint = endpoint;
+  response.userId = message.userId || message.anonymousId;
+  response.body.JSON = removeUndefinedAndNullValues(payload);
+  return {
+    ...response,
+    headers: {
+      "Content-Type": "application/json",
+    }
+  };
+}
+
+function getIdentifyPayload(message) {
+  let payload = {};
```

Comment:
Can we convert it to a config JSON and use it along with `constructPayload`?

Reply:
Yes, Converted and used it along with **constructPayload**